tests/test_round3_harness.py

```
@@ -0,0 +1,54 @@
+    Path(__file__).resolve().parent.parent
+    / "experiments"
+    / "prompt-injection-boundary-tags"
+    / "rounds"
+    / "round3"
+    / "harness"
+    / "run_experiment.py"
+)
+
+spec = importlib.util.spec_from_file_location("round3_harness", MODULE_PATH)
+if spec is None or spec.loader is None:
+    raise RuntimeError(f"Failed to load module spec from {MODULE_PATH}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)